solidity_contracts/src/CairoCounterCaller.sol

```diff
@@ -7,7 +7,7 @@ using CairoLib for uint256;
 
 contract CairoCounterCaller  {
     /// @dev The cairo contract to call
-    uint256 cairoCounter;
+    uint256 immutable cairoCounter;
```

Comment:
I think a comment on why the `cairoCounter` is a `uint256` should be added.
It could be confusing for a solidity dev to call a `uint256` instead of an `address`